src/stores/cluster/index.js

```diff
@@ -55,6 +53,14 @@ export default class ClusterStore extends Store {
 
   store = {};
 
+  get clusterEnv() {
```

Comment:
If clusterEnv changed, should update component ?

You can add @computed from mobx